karma.conf.js

```diff
@@ -56,12 +53,12 @@ module.exports = function (config) {
 
     // start these browsers
     // available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
-    browsers: ['Chrome', 'Firefox'],
+    browsers: !!process.env.TEST_ON_FIREFOX ? ['Firefox'] : ['ChromeHeadless'],
 
 
     // Continuous Integration mode
     // if true, Karma captures browsers, runs the tests and exits
-    singleRun: false,
+    singleRun: true,
```

Comment:
I tackled the CI vs local like this: https://github.com/RedHatInsights/policies-ui-frontend/blob/master/config/karma.conf.ts
This is already good, just pointing other ways to do it :)

Reply:
Indeed good ideas, I keep it for the next time I will face that, and I didn't know about the FirefoxHeadless :)